query/README.md

```diff
@@ -1,66 +1,203 @@
 # [MLIR](https://mlir.llvm.org) + [hail](https://hail.is) = 🚀🧬?
 
-## Dependencies
+## Installing build tools
 
-You will need `cmake` and `ninja`. If you're on OS X, use https://brew.sh:
+You will need to install `cmake` and `ninja`. If you're on macOS, you can do so
+using [Homebrew](https://brew.sh):
 
 ```sh
 brew install cmake ninja
 ```
 
-## Building/Installing LLVM and MLIR
-Obviously, update the paths for your environment. `$HAIL_DIR` is the root of the
-hail repository.
+## Setting up your shell environment
+
+In your shell configuration file (e.g. `~/.bashrc` for `bash`), add the following,
+and update variable values where specified:
+
+```sh
+# replace with the path to your local clone of the Hail repository
+export HAIL_DIRECTORY=$HOME/hail
```

Comment:
Objections to calling this `HAIL` instead? We use that elsewhere in the dev-docs.

Reply:
Dropped `_DIRECTORY` from the ends of these variables!